docs/apm/filters.asciidoc

```diff
@@ -46,3 +54,14 @@ v|*Go:* {apm-go-ref}/configuration.html#config-environment[`ELASTIC_APM_ENVIRONM
 *Ruby:* {apm-ruby-ref}/configuration.html#config-environment[`environment`]
 *Real User Monitoring:* {apm-rum-ref}/configuration.html#environment[`environment`]
 |===
+
+[[local-filters]]
+==== Local filters
+
+Local filters are ways you can filter your APM data on each individual page.
+They do not persist as you move between pages.
```

Comment:
They do persist between pages, if applicable. E.g. if you select an host on the services overview, then click on a transaction, the host is still selected.

Reply:
This is something that was not part of an initial implementation, but it should be there in 7.4 .

Reply:
Ahhh I see. Some do and some don't. It depends on the filter. Thanks for pointing this out.

Reply:
I've updated the copy, but I wonder if it still makes sense to think of these as "global" and "local" filters? Do we have another name that we've been calling these new filters?

Reply:
@bmorelli25 Global / local feels very much like an internal way to describe the placement on the filter options. Perhaps the global ones can be better described as queries (search, time and environment) whereas the local filters are contextual to the page and the data. I'm sure there's a better way to describe it...

Reply:
We've been calling it local UI filters, but that doesn't help a lot. I think the distinction is that global filters are filters that are available on all pages / data, and local filters are specific to the page / data you are currently viewing.

Reply:
I think how @dgieselaar just described is quite accurate to understand the difference when they're compared on a page like this.

Reply:
Great stuff - thanks!